recipes/openldap/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+required_conan_version = ">=1.43.0"
+
+
+class OpenldapConan(ConanFile):
+    name = "openldap"
+    description = "OpenLDAP C++ library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.openldap.org/"
+    license = "OLDAP-2.8"
+    topics = ("ldap", "load-balancer", "directory-access")
+    exports_sources = ["patches/*"]
+    settings = settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_cyrus_sasl": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_cyrus_sasl": True
+
+    }
+    _autotools = None
+    _configure_vars = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self._source_subfolder)
+
+    def requirements(self):
+        self.requires("openssl/1.1.1m")
```

Comment:
```suggestion
        self.requires("openssl/1.1.1n")
```
Between the interval of your first commit and the last, a new security patch of OpenSSL was released